### PR TITLE
feat: add fennel_language_server

### PR DESCRIPTION
**Pull request by Massolari**

### CODE REVIEW

**Comment by glepnir** [comment]
does this server not work on windows?

**Comment by Massolari** [comment]
@glepnir I don't know. I only tested it on MacOS

**Comment by glepnir** [comment]
nope but it would be better add the windows env.

File: lua/lspconfig/server_configurations/fennel_language_server.lua [hunk 1]
```diff
@@ -0,0 +1,24 @@
+local util = require 'lspconfig.util'
+
+local cmd = { 'fennel-language-server' }
+
+if vim.fn.has 'win32' == 1 then
```
Comment (Massolari) [inline lua/lspconfig/server_configurations/fennel_language_server.lua:5]: @glepnir Done!
Is it correct?

Comment (glepnir) [reply lua/lspconfig/server_configurations/fennel_language_server.lua:5]: yes

**Pull request merged**